apps/web/src/pages/integrations/components/IntegrationInput.tsx

```diff
@@ -46,6 +46,30 @@ export function IntegrationInput({
     );
   }
 
+  if (credential.type === 'switch') {
+    console.log(field);
+
+    return (
+      <Switch
+        styles={() => ({
+          root: {
+            display: 'block !important',
+            maxWidth: '100% !important',
```

Comment:
Do we have any other way of implement the style without using `!important`?

Reply:
no this is what I know the closest way to do it and it did not work without it 😢